sllm_store/examples/load_quantized_transformers_model.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import time
+
+import torch
+
+from transformers import AutoTokenizer, BitsAndBytesConfig
+
+from sllm_store.transformers import load_model
+
+parser = argparse.ArgumentParser(description="Load a model from ServerlessLLM")
+parser.add_argument(
+    "--model_name", type=str, required=True, help="Model name stored"
+)
+parser.add_argument(
+    "--storage_path",
```

Comment:
We now use hyphens (-) instead of underscores (_), so please update this for consistency.